netlify.toml

```
@@ -0,0 +1,12 @@
+# Production context:
+# All deploys from the main repository branch
+# will inherit these settings.
+[context.production]
+  command = "bundle exec jekyll build"
+
+# Deploy Preview context:
+# All deploys generated from a pull/merge request
+# will inherit these settings.
+[context.deploy-preview.environment]
```

```suggestion
[context.deploy-preview]
```
The suffix `.environment` is for defining context specific environment variables. Isn't it?

See the sample file in [the docs](https://docs.netlify.com/configure-builds/file-based-configuration/#sample-file).

---

true!